test_env_vars.yaml

```diff
@@ -118,3 +118,22 @@ various_packages:
     repo: https://github.com/akhmanova/cachito-gomod-test
     ref: 1827221d787cbd1e979b339cfbbf59728eddf0d4
     dependencies_count: 7
+# Test data for pip packages
+pip_packages:
+  # pip package without dependencies and requirements.txt
+  # repo: The URL for the upstream git repository
+  # ref: A git reference at the given git repository
+  # package: expected package from the Cachito response
```

Comment:
Why did you remove `expected_files`? That one is still relevant, right?

Reply:
OMG, sorry for it.

Reply:
no problem :-)